lib/openscap_parser/rule.rb

```diff
@@ -57,15 +67,23 @@ def identifier_node
       @identifier_node ||= parsed_xml.at_xpath('ident')
     end
 
+    def parent_id
+      @parent_id = parsed_xml.xpath("name(..)='Group'") ?
```

Comment:
Same here, does this handle polymorphism? The parent also can be the `Profile` itself, not just a `Group`...